client/components/mma/cancel/CancellationReasonReview.tsx

```diff
@@ -323,26 +341,34 @@ const ConfirmCancellationAndReturnRow = (
 					>
 						<Button
 							icon={
-								showOfferBeforeCancelling === 'pending' ? (
+								showAlternativeBeforeCancelling ===
+								'pending' ? (
 									<SvgSpinner size="xsmall" />
 								) : (
 									<SvgArrowRightStraight />
 								)
 							}
 							iconSide="right"
-							disabled={showOfferBeforeCancelling === 'pending'}
+							disabled={
+								showAlternativeBeforeCancelling === 'pending'
+							}
 							aria-disabled={
-								showOfferBeforeCancelling === 'pending'
+								showAlternativeBeforeCancelling === 'pending'
 							}
 							onClick={() => {
 								if (props.onClick) {
 									props.onClick();
 								}
-								if (showOfferBeforeCancelling) {
-									navigate('../offer', {
+								if (showAlternativeBeforeCancelling) {
+									const cancelAlternativeUrlPart =
+										cancelAlternativeUrlPartLookup[
+											productType.productType
+										] || '';
```

Comment:
yep ... there's a chance this wou'd still take you to a valid url with the extra `|| ''` as oppsed to cancelAlternativeUrlPart being undefined which would lead the router to navigate to `../undefined` ... we should never get into this state though and could handle this error another way